packages/vector_graphics_compiler/lib/src/svg/path_ops.dart

```diff
@@ -0,0 +1,390 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
```

Comment:
Please add a comment about where this was copied from and that it should be kept in sync with that source for now.

Reply:
Added.